utils/generateImplementedTestCases.pl

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env perl
+
+use 5.16.0;
+use warnings;
+use Zonemaster::Engine;
+use File::Basename;
+
+# page header
+print '<!-- File generated by ', basename ($0), ", script in zonemaster/zonemaster util directory.\n";
+print "Use that script to generate a new file for each release of Zonemaster when \n";
+print "Zonemaster-Engine also has been updated.-->\n\n";
+print "# Implemented Test Cases\n\n";
+print "Index of Text Case specifications is also found in [README](README.md).\n";
+print "[Zonemaster-Engine] is the repository of the implementation of the Test Cases (the methods).\n";
+print "\n\n";
+
+# table header
+print "| Module | Method (implemented test case) | Test Case Specification |\n";
```

Comment:
IMO we shouldn't be referring to test cases as "methods" in the specifications.

Reply:
"Method" here refer to the Perl code.

Reply:
At least it's (locally) consistent that way. Though this is part of the specification, so I think it would be more reasonable to have it speak the language of the specification.

Also, we should be able to change the implementation without invalidating the specification. E.g. we should be able to switch programming languages to Python or Haskell, or switch from OO-style Perl to procedural style Perl.

I think the appropriate terminology here would be:
```
| Test Level | Test Case Identifier | Test Case Specification |
```

Though we seem to be using "Test Level" and "Test Plan" kind of interchangeably, with "Test Plan" begin more common. So maybe "Test Plan" would be better than "Test Level" at this point (even though I believe it to be less technically correct).

Reply:
> At least it's (locally) consistent that way. Though this is part of the specification, so I think it would be more reasonable to have it speak the language of the specification.

This is located with the specification, but links to the implementation in Perl (the first column). The second column has the name of the Perl subroutine (method) that implements the test case. The third column has the test case identifier and a link to the specification.

> Also, we should be able to change the implementation without invalidating the specification. E.g. we should be able to switch programming languages to Python or Haskell, or switch from OO-style Perl to procedural style Perl.

In that case this would just be a small part of the updates we would have to do.